src/SystemCommands-VariableCommands/SycGenerateVariableAccessorsWithLazyInitializationCommand.class.st

```diff
@@ -0,0 +1,57 @@
+Class {
+	#name : #SycGenerateVariableAccessorsWithLazyInitializationCommand,
+	#superclass : #CmdCommand,
+	#instVars : [
+		'variable',
+		'value'
+	],
+	#category : #'SystemCommands-VariableCommands'
+}
+
+{ #category : #testing }
+SycGenerateVariableAccessorsWithLazyInitializationCommand class >> canBeExecutedInContext: aToolContext [
+	^aToolContext isVariableSelected and: [ aToolContext isGlobalVariableSelected not ]
+]
+
+{ #category : #activation }
+SycGenerateVariableAccessorsWithLazyInitializationCommand class >> fullBrowserMenuActivation [
+	<classAnnotation>
+	
+	^CmdContextMenuActivation byRootGroupItemOrder: 2 for: ClyFullBrowserVariableContext 
+]
+
+{ #category : #converting }
+SycGenerateVariableAccessorsWithLazyInitializationCommand >> asRefactorings [
+	^ { (RBCreateAccessorsWithLazyInitializationForVariableRefactoring 
+		variable: variable name
+		class: variable definingClass 
+		classVariable: variable definingClass isMeta
```

Comment:
I think isMeta is deprecated by isClassSide

Reply:
Hello @dionisiydk 

Yes, I made this mistake because I got confused with RBClass. I fixed this bug in this PR: https://github.com/pharo-project/pharo/pull/9265